staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5891,6 +5893,15 @@ type NodeRuntimeHandler struct {
 	Features *NodeRuntimeHandlerFeatures `json:"features,omitempty" protobuf:"bytes,2,opt,name=features"`
 }
 
+// NodeFeatures describes the set of implemented features implemented by the CRI implementation.
+// THE FEATURES CONTAINED IN THE NodeFeatures SHOULD DEPEND ON ONLY CRI IMPLEMENTATION, BE INDEPENDENT ON RUNTIME HANDLERS,
```

Comment:
same

Reply:
nit: "independent of" rather than "on"

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/125470/commits/ca414b0f387753d8a6d4e77ec3563eb657e11f59

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/125470/commits/ca414b0f387753d8a6d4e77ec3563eb657e11f59